BUCK

```diff
@@ -1,10 +1,25 @@
+genrule(
+  name = "chromium-build-image",
+  srcs = ["Dockerfile.build"],
+  remote = False,
+  cmd = "docker build -t chromium-build-new - < Dockerfile.build && echo chromium-build-new > $OUT",
```

Comment:
This is also gross. The more right way to do this is to export the built Docker image as a `.tar` and then when we build Chromium `docker load` that tar file.

Unfortunately, the image is several GB large and exporting it to the disk + importing it adds a lot of time to the build.